gulp/tasks/build.js

```diff
@@ -172,13 +166,13 @@ function compileIndvES2015ModulesToBrowser() {
         },
         footer: fileName => {
           return isFirebaseApp(fileName) ? `
-          })();` : `
+          })().default;` : `
```

Comment:
Originally I was using `babel-plugin-add-module-exports` for that reason. If we are going to move away from using the babel-loader then we need to do something to recreate that.